### PR TITLE
Increase size of checkbox, switch, and radio button to 48 by 48

### PR DESCRIPTION
**Pull request by jonahwilliams**
Conform to android a11y scanner and 48 by 48 tap target recommendation.

### CODE REVIEW

**Comment by jonahwilliams** [comment]
This is a breaking change, so i will go through the normal process before submitting

**Comment by goderbauer** [comment]
This will only increase the hit box, right? Not the visual appearance of the checkbox/switch/radio button? Might be worth to be specific in the title for future archeologists.

File: packages/flutter/lib/src/material/checkbox.dart [hunk 1]
```diff
@@ -200,7 +200,7 @@ class _RenderCheckbox extends RenderToggleable {
         activeColor: activeColor,
         inactiveColor: inactiveColor,
         onChanged: onChanged,
-        size: const Size(2 * kRadialReactionRadius, 2 * kRadialReactionRadius),
+        size: const Size(2 * kRadialReactionRadius + 8.0, 2 * kRadialReactionRadius + 8.0),
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/checkbox.dart]: Is there value in maybe adding a comment to explain where the 8.0 is coming from here and in other places?

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/material/checkbox.dart (hunk 1 above)
Comment (HansMuller) [reply packages/flutter/lib/src/material/checkbox.dart]: The + 8.0 serves to increase the radius from 20 to 24. We should not increase the radius from 20 to 24.

**Comment by jonahwilliams** [comment]
Closing in favor of https://github.com/flutter/flutter/pull/18369

**Pull request closed without merging**